### ISSUE TITLE
internal server error when trying to syncronize all PRs

### ISSUE THREAD

**Issue by ignatenkobrain** [servo/homu#49]
```
Error: 500 Internal Server Error

Sorry, the requested URL 'http://ci.dnf.baseurl.org:54856/callback?code=xxx&state=%7B%22cmd%22%3A%22synch%22%2C%22repo_label%22%3A%22all%22%7D' caused an error:

Internal Server Error
```

We have 2 repos in config: `dnf` and `dnf-plugins-core`, so I opened `/queue/all` and clicked syncronize PRs and it failed.

**Comment by Manishearth** [comment]
Yeah, this is because /all is special. We need to special case it, basically.

**Comment by Manishearth** [comment]
barosl didn't want to do this because on the main homu instance people shouldn't have access to all queues, but for individual deployments like we have this is okay. Basically, https://github.com/servo/homu/blob/master/homu/server.py#L636 needs to handle "all" specially.
